tools/oss-fuzz.sh

```diff
@@ -32,15 +32,15 @@ meson_args=("-Db_lundef=false")
 if [ -z "$FUZZING_ENGINE" ]; then
     meson_args+=("-Dllvm-fuzz=true")
 else
-    meson_args+=("-Doss-fuzz=true" "--auto-features=disabled" "-Dnspawn=enabled" "-Dresolve=true")
```

Comment:
The runtime environment on OSS-Fuzz expects dependencies to be built and linked against the fuzz targets by the build script one way or another: https://google.github.io/oss-fuzz/further-reading/fuzzer-environment/#runtime-dependencies and that's why `--auto-features=disabled` is used. `libcap`, `libmount` and so on happen to be there by accident but it's still broken in the sense that the fuzz targets produce false positives under MSan because those libraries aren't instrumented.

Reply:
Ugh, what a mess

Reply:
There were attempts to make it less messy (and more useful) but it didn't go anywhere so it is what it is unfortunately.

Reply:
OK we now build fuzz-journal-remote unconditionally again

Reply:
FWIW I think it would be better to add https://google.github.io/oss-fuzz/further-reading/fuzzer-environment/#runtime-dependencies to the comment. The dependencies don't have to be linked statically. They can be linked dynamically (and all the fuzz targets are linked dynamically against libsystemd-shared for example)